app/models/user.rb

```diff
@@ -55,7 +55,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :invitable, :database_authenticatable, :registerable, :uid,
          :recoverable, :rememberable, :trackable, :validatable, :omniauthable,
-         omniauth_providers: [:google_oauth2]
+         omniauth_providers: %i[google_oauth2 facebook]
```

Comment:
A suggestion for increased maintainability, could be a good idea to rename the callback `facebook` to `facebook_oauth`.

Reply:
Sure I'll do this